stacks/tcpconn.go

```diff
@@ -160,8 +161,10 @@ func (sock *TCPConn) Write(b []byte) (n int, _ error) {
 	}
 }
 
-// Read reads data from the socket's input buffer. If the buffer is empty,
-// Read will block until data is available.
+
+// Read reads data from the socket's input (RX) (ring) buffer... populating b[].. 
+//If the rx buffer is empty, Read will block until data is available.
+
```

Comment:
Please follow `//` with a space and make it adjacent to function name so it appears as part of documentation.